Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/Display/XamlRenderer.cs

```diff
@@ -999,24 +1005,33 @@ private void RenderItalicRun(InlineCollection inlineCollection, ItalicTextInline
         /// <param name="context"> Persistent state. </param>
         private void RenderStrikethroughRun(InlineCollection inlineCollection, StrikethroughTextInline element, RenderContext context)
         {
-            Span span = new Span
+            Span span = new Span();
+
+            if (TextDecorationsSupported)
             {
-                FontFamily = new FontFamily("Consolas")
-            };
+                span.TextDecorations = TextDecorations.Strikethrough;
+            }
+            else
+            {
+                span.FontFamily = new FontFamily("Consolas");
+            }
 
             // Render the children into the inline.
             RenderInlineChildren(span.Inlines, element.Inlines, span, context);
 
             AlterChildRuns(span, (parentSpan, run) =>
```

Comment:
AlterChildRuns is not going to do anything with the if inside, but there is no reason to iterate it to do nothing. Just check if TextDecorationsSupported is not true before calling the method, and only then iterate if needed.